### PR TITLE
finagle v7 release post

### PR DESCRIPTION
**Pull request by dschobel**

### CODE REVIEW

File: source/blog/2017-08-17-release-notes-7-00.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### Scrooge ###
+
+* scrooge-generator: Generated Java code now is using `c.t.s.TReusableBuffer` to reduce
+  object allocations. [4cd9ee7](https://github.com/twitter/scrooge/commit/4cd9ee7017d75cec068f5acf14b97bc2955474ec)
+* scrooge-generator: Generated Cocoa code now supports modular frameworks and 
+  removes some compiler warnings about implicit casts. [f48cd56](https://github.com/twitter/scrooge/commit/f48cd56e9398305ee7e280e161b0423da4131845)
+  
+### Util ###
+Util's StatsReceivers learned about Vebosity Levels as a way to reduce the number of exported metrics during normal operations.
```
Comment (mosesn) [inline source/blog/2017-08-17-release-notes-7-00.md]: s/vebosity/verbosity

File: source/blog/2017-08-17-release-notes-7-00.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Without further ado, here are the highlights:
+
+[Finagle 7.0.0][finagle], [Finatra 2.12.0][finatra], [Scrooge 4.19.0][scrooge], [TwitterServer 1.31.0][twitterserver], and [Util 7.0.0][util].
+
+
+### Finagle ###
+
+* finagle-core: Finagle is now fully decoupled from Netty 3. [49d4d0d](https://github.com/twitter/finagle/commit/49d4d0df6d7e0a5dcac85ca6564efcae101c8e04)
+* finagle-mux: Introduce to new more efficient message decoder. [f432bd4](https://github.com/twitter/finagle/commit/f432bd443c4998431e4cbfeb6934916e7310c9a8)
```
Comment (mosesn) [inline source/blog/2017-08-17-release-notes-7-00.md]: s/Introduce to new/Introduce a new

File: source/blog/2017-08-17-release-notes-7-00.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+layout: post
+title: Finagle 7 Release Notes
+published: true
+post_author:
+  display_name: Daniel Schobel
+  twitter: dschobel
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+The August release is here and brings a major version bump for both Finagle and Util to v7! :cake::cake::cake:
```
Comment (yufangong) [inline source/blog/2017-08-17-release-notes-7-00.md]: :cake::cake::cake:

File: source/blog/2017-08-17-release-notes-7-00.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Over the last few years Finagle has had an extensive internal overhaul to support Netty 4 along with a rewrite of all its protocols so now as the last traces of Netty 3 have been removed from finagle-core we felt the time was right to celebrate with a shiny new major version.
+
+(Fun fact: Finagle's last major version bump came almost 5 years ago, a simpler time when 'Gangnam Style' was still dominating the charts.)
+
+Without further ado, here are the highlights:
+
+[Finagle 7.0.0][finagle], [Finatra 2.12.0][finatra], [Scrooge 4.19.0][scrooge], [TwitterServer 1.31.0][twitterserver], and [Util 7.0.0][util].
+
+
+### Finagle ###
```
Comment (vkostyukov) [inline source/blog/2017-08-17-release-notes-7-00.md]: I think we should also mention that finagle-stats is no longer depending on Commons (big deal).

Comment (vkostyukov) [inline source/blog/2017-08-17-release-notes-7-00.md]: What do you think about mentioning Tunables being available for users?

**Pull request merged**